pages/documentation/developer/api_guide/single-line-diagram_svg-writing.md

```diff
@@ -0,0 +1,214 @@
+We are showing in this guide how to create some single line diagrams
+- first from a test network,
+- then from a CGMES file.
+
+*Note: the 1.8.0-SNAPSHOT version of powsybl-single-line-diagram was used to generate the SVGs of this guide. Because of that, you might experience some differences with the examples shown, which will be corrected by 1.8.0 release (expected: 2020-11-26).*
+
+
+## Prerequisites
+
+### Maven dependencies
+First of all, we need to add some Maven dependencies in our `pom.xml` file:
+
+```xml
+<dependencies>
+    <dependency>
+        <groupId>com.powsybl</groupId>
+        <artifactId>powsybl-single-line-diagram-core</artifactId>
+        <version>${powsybl.sld.version}</version>
+    </dependency>
+    <dependency>
+        <groupId>com.powsybl</groupId>
+        <artifactId>powsybl-iidm-impl</artifactId>
+        <version>${powsybl.core.version}</version>
+    </dependency>
+    <dependency>
+        <groupId>com.powsybl</groupId>
+        <artifactId>powsybl-iidm-test</artifactId>
+        <version>${powsybl.core.version}</version>
+    </dependency>
+    <dependency>
+        <groupId>com.powsybl</groupId>
+        <artifactId>powsybl-config-test</artifactId>
+        <version>${powsybl.core.version}</version>
+    </dependency>
+    <dependency>
+        <groupId>com.powsybl</groupId>
+        <artifactId>powsybl-cgmes-conversion</artifactId>
+        <version>${powsybl.core.version}</version>
+    </dependency>
+    <dependency>
+        <groupId>com.powsybl</groupId>
+        <artifactId>powsybl-triple-store-impl-rdf4j</artifactId>
+        <version>${powsybl.core.version}</version>
+    </dependency>
+    <dependency>
+        <groupId>org.slf4j</groupId>
+        <artifactId>slf4j-simple</artifactId>
+        <version>${slf4j.version}</version>
+    </dependency>
+</dependencies>
+
+<properties>
+    <powsybl.sld.version>1.7.2</powsybl.sld.version>
+    <powsybl.core.version>3.7.1</powsybl.core.version>
+    <slf4j.version>1.7.22</slf4j.version>
+</properties>
+```
+
+Here are some details about these dependencies:
+- `powsybl-single-line-diagram-core` is the core module of single-line-diagram,
+- `powsybl-iidm-impl` is for the Network model,
+- `powsybl-iidm-test` is for loading the test Network,
+- `powsybl-config-test` is for loading the test configuration (see [configuration API guide](configuration.md)),
+- `powsybl-cgmes-conversion` and `powsybl-triple-store-impl-rdf4j`  are for importing a CGMES file,
+- `slf4j-simple` allows you to have simple logging capabilities.
+
+### Layout parameters
+For both example networks, we need to initialize a few layout parameters before generating diagrams (see [SVG layouts]() API guide for more info):
```

Comment:
Not sure to understand what you mean by
> For both example networks

`For both examples` or `For both networks` seems sufficient?

I would also change the end to:
> see [SVG layouts guide]() for more info